networkx/algorithms/tests/test_cycles.py

```diff
@@ -863,3 +866,36 @@ def test_complete_graph(self):
     def test_tree_graph(self):
         tg = nx.balanced_tree(3, 3)
         assert not minimum_cycle_basis(tg)
+
+
+class TestGirth:
+    @classmethod
+    def setup_class(cls):
+        T = nx.Graph()
+        nx.add_cycle(T, [1, 2, 3, 4], weight=1)
+        T.add_edge(2, 4, weight=5)
+        cls.diamond_graph = T
+
+    def test_chvatal_graph(self):
+        graph = nx.chvatal_graph()
+        assert girth(graph) == 4
+
+    def test_tutte_graph(self):
+        graph = nx.tutte_graph()
+        assert girth(graph) == 4
+
+    def test_petersen_graph(self):
+        graph = nx.petersen_graph()
+        assert girth(graph) == 5
+
+    def test_heawood_graph(self):
+        graph = nx.heawood_graph()
+        assert girth(graph) == 6
+
+    def test_pappus_graph(self):
+        graph = nx.pappus_graph()
+        assert girth(graph) == 6
+
+    def test_random_tree(self, n=100):
+        graph = nx.random_tree(n)
```

Comment:
To ensure our tests don't have intermittent failures (which are really hard to track down), we add "seed" values for functions in tests that use random number generators.  Can you add something like:  `seed=42` to the `random_tree` call?

Reply:
Done!